examples/recording_service/service_admin/cpp/CMakeLists.txt

```diff
@@ -10,7 +10,8 @@
 #  use or inability to use the software.
 #
 cmake_minimum_required(VERSION 3.11)
-project (RecorderRemoteTimeTagging)
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
Apply this as a target property. https://cmake.org/cmake/help/latest/command/set_target_properties.html